dotCMS/src/main/java/com/dotcms/rest/api/v1/browser/BrowserQueryForm.java

```diff
@@ -0,0 +1,262 @@
+package com.dotcms.rest.api.v1.browser;
+
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.dotmarketing.util.UtilMethods;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonDeserialize(builder = BrowserQueryForm.Builder.class)
+public class BrowserQueryForm {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class cannot be instantiated and does not provide any static methods or fields](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5337824)